image-loader/app/lib/FailureResponse.scala

```diff
@@ -54,6 +54,16 @@ object FailureResponse extends ArgoHelpers with Results {
     )
   }
 
+  def imageAlreadyExists(exception: Exception): Response = {
+    logger.info(s"Rejected request to load file: image file is already exists and soft deleted", exception)
```

Comment:
Should this read "... image file already exists and is soft deleted" instead?

Reply:
`image file  already exists in a deleted state` ?